Godeps/Godeps.json

```diff
@@ -3043,6 +3043,10 @@
 			"ImportPath": "google.golang.org/api/pubsub/v1",
 			"Rev": "c0dae069ee96c9261a04c81efd9e0f1e55f565ac"
 		},
+		{
+			"ImportPath": "google.golang.org/api/tpu/v1alpha1",
```

Comment:
Let's switch to v1beta1

Reply:
We don't have generated clients for v1beta1 at this point. I asked the TPU team to generate and publish them on https://github.com/google/google-api-go-client.